packages/ingest/granule.js

```diff
@@ -551,6 +551,7 @@ class Granule {
         await aws.deleteS3Object(bucket, stagedFileKey);
       }
       else {
+        log.debug('Checksum different, renaming file');
```

Comment:
Checksum of what file?  This message could use more detail in order to be helpful.